src/options/parallel_options.toml

```diff
@@ -69,3 +80,12 @@ name   = "Parallel"
   default    = "ManagedOut()"
   includes   = ["<iostream>", "options/managed_streams.h"]
   help       = "set the output channel for writing partitions"
+
+[[option]]
+  name       = "emitLearnedLiterals"
+  category   = "expert"
+  long       = "emit-learned-literals"
```

Comment:
Can we change the name of this option to something parallel specific? As is, it may be confused with e.g. `--produce-learned-literals`.